srcpkgs/pam-poldi/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'pam-poldi'
+pkgname=pam-poldi
+version=0.4.2
+revision=1
+# the git-hash of v0.4.2 as it isn't tagged
```

Comment:
please ask upstream to tag it

Reply:
I'm not quite sure why it's not tagged, package developer also does packaging for debian and he uses non-tagged code: https://packages.debian.org/stable/libpam-poldi. I'll email him and ask if it will be possible to tag it.